content/blog/deploying-mongodb-application-on-hpe-greenlake-for-containers.md

```diff
@@ -0,0 +1,948 @@
+* In the Create Cluster form, provide the cluster name 'hpe', and select the standard cluster blueprint. The new cluster appears in the list of clusters.
+
+![](/img/image-8.png)
+
+
+
+![](/img/image-9.png)
+
+As indicated above there are multiple clusters deployed in parallel for multiple purposes. For MongoDB Application deployment we will use cluster created with name 'hpe'.
+
+![](/img/image-10.png)
+
+### Step-2: Download scoped kubeconfig from Container platform page
+
+* From the Clusters tab, select the 'hpe' Kubernetes cluster and click Launch Service Console. You are directed to the container platform page.
+* Click on Download kubeconfig.
+
+![](/img/image-11.png)
+
+
+
+![](/img/image-12.png)
+
+Note: Launching HPE Ezmeral Runtime Enterprise from HPE GreenLake Central is configured through SAML SSO and adds a session token to the kubeconfig file. You will need to download the kubeconfig file again if you want to continue to access the cluster when the session token expires after an hour. 
+
+### Step-3: View the 'hpe' Kubernetes cluster environment details
+
+View the Kubernetes version
+
+```shellsession
+$ kubectl version --short
+Client Version: v1.20.0
+Server Version: v1.20.11-hpe-2
+```
+
+View the deployed control plane nodes and worker nodes
+
+```shellsession
+$ kubectl get nodes -o wide
+NAME                                        STATUS   ROLES                  AGE   VERSION          INTERNAL-IP     EXTERNAL-IP   OS-IMAGE                              KERNEL-VERSION                 CONTAINER-RUNTIME
+k8s-hpe-master-d6xv8-254v7.glhc-hpe.local   Ready    control-plane,master   71d   v1.20.11-hpe-2   172.16.17.115   <none>        SUSE Linux Enterprise Server 15 SP2   5.3.18-150200.24.115-default   containerd://1.5.1-hpe-1
+k8s-hpe-master-d6xv8-8fxxz.glhc-hpe.local   Ready    control-plane,master   71d   v1.20.11-hpe-2   172.16.17.110   <none>        SUSE Linux Enterprise Server 15 SP2   5.3.18-150200.24.115-default   containerd://1.5.1-hpe-1
+k8s-hpe-master-d6xv8-jjrpc.glhc-hpe.local   Ready    control-plane,master   71d   v1.20.11-hpe-2   172.16.17.114   <none>        SUSE Linux Enterprise Server 15 SP2   5.3.18-150200.24.115-default   containerd://1.5.1-hpe-1
+k8s-hpe-worker-qscr4-89n67.glhc-hpe.local   Ready    worker                 71d   v1.20.11-hpe-2   172.16.17.109   <none>        SUSE Linux Enterprise Server 15 SP2   5.3.18-150200.24.115-default   containerd://1.5.1-hpe-1
+k8s-hpe-worker-qscr4-fp8px.glhc-hpe.local   Ready    worker                 71d   v1.20.11-hpe-2   172.16.17.116   <none>        SUSE Linux Enterprise Server 15 SP2   5.3.18-150200.24.115-default   containerd://1.5.1-hpe-1
+k8s-hpe-worker-qscr4-l95j4.glhc-hpe.local   Ready    worker                 71d   v1.20.11-hpe-2   172.16.17.113   <none>        SUSE Linux Enterprise Server 15 SP2   5.3.18-150200.24.115-default   containerd://1.5.1-hpe-1
+```
+
+### Step-4: View the 'hpe' Kubernetes cluster default storage class configuration
```

Comment:
Similarly, before putting in this screen shot, you may want a sentence just below "Step-4" that talks about what you're instructing them to do.
But I have a question... could Step-4 be a 3rd bullet in Step-3? Or does it really need to stand alone? If you decided to make it a 3rd bullet in Step-3, don't forget to renumber your other steps... Step 5 becomes Step 4, etc..

Reply:
Good point, clubbed step3 and step4. and re ordered the numbering. Done.